src/cloffeine/async_loading_cache.clj

```diff
@@ -1,48 +1,58 @@
 (ns cloffeine.async-loading-cache
   (:refer-clojure :exclude [get])
-  (:require [cloffeine.common :as common]
-            [cloffeine.async-cache :as acache]
+  (:require [cloffeine.async-cache :as acache]
```

Comment:
another one. Is this the IDE sorting the lines alphabetically?

Reply:
Commented above.